Description: The following test assumes that the locale "pt_BR.UTF-8" is
installed, and fails otherwise. Since there is no "non-trivial" locale which we
can assume is installed, the only way to fix this seems to disable it.
Forwarded, but upstream seems willing to not drop the test.
Author: Pietro Battiston <toobaz@email.it>
Forwarded: yes

--- python-shapely.orig/shapely/tests/wkt_locale.txt	2010-09-14 11:43:00.000000000 +0200
+++ /dev/null	1970-01-01 00:00:00.000000000 +0000
@@ -1,28 +0,0 @@
-Test locale independence of WKT
-===============================
-
-Set locale to one that uses a comma as decimal seperator`
-
-  >>> import locale, sys
-  >>> if sys.platform == 'win32':
-  ...     _ = locale.setlocale(locale.LC_ALL, 'portuguese_brazil')
-  ... else:
-  ...     _ = locale.setlocale(locale.LC_ALL, 'pt_BR.UTF-8')
-
-Test reading and writing`
-
-  >>> from shapely.wkt import loads
-  >>> p = loads('POINT (0.0 0.0)')
-  >>> p.x == 0.0
-  True
-  >>> p.y == 0.0
-  True
-  >>> p.wkt
-  'POINT (0.0...)'
-  >>> from shapely.wkt import dumps
-  >>> dumps(p)
-  'POINT (0.0...)'
-
-Reset locale
-
-  >>> _ = locale.setlocale(locale.LC_ALL, '')
